middleware/kubernetes/kubernetes.go

```diff
@@ -432,49 +448,60 @@ func symbolContainsWildcard(symbol string) bool {
 //   * Fist two segments must start with an "_",
 //   * Second segment must be one of _tcp|_udp|_*|_any
 func ValidSRV(name string) bool {
+	// First label
+	first_ok := false
+
+	if  len(name) > 0 && name[0] == '_' {
```

Comment:
It needs to be changed to accept wildcard fields, which now do not have the SRV field underscore prefix.
Not allowing/requiring the standard SRV underscore prefixes for the SRV fields when the values are wild cards means we have to special-case the wildcard checks.